docs/tutorials/create_architecture.md

```diff
@@ -110,6 +110,7 @@ path:
 
 ```python
 arch.add_path('target_path', path)
+! rm ./architecture.yaml
 arch.export('./architecture.yaml')
```

Comment:
How about replacing this line by the following code?
```suggestion
arch.export('./architecture.yaml', force=True)
```

Reply:
Thank you for your review. I've commited your suggestion.